ForMiniNTuple/lib/MNT_Helper.py

```diff
@@ -105,11 +155,69 @@ def FillHistTerm(histos, term, signals, bkgs, value, Sample_ID, event_wgt):
         histos[term]["ZZ"].Fill(value, event_wgt)
     elif Sample_ID == -2424: # WW
 	histos[term]["WW"].Fill(value, event_wgt)
-    elif Sample_ID/10000 < -23:   # triboson
+    elif Sample_ID == -242424 or Sample_ID == -242423 or Sample_ID == -242323 or Sample_ID == -232323:   # triboson
         histos[term]["triboson"].Fill(value, event_wgt)
     elif Sample_ID == -062300:
         histos[term]["tZq"].Fill(value, event_wgt)
     elif Sample_ID == -060623:
         histos[term]["ttZ"].Fill(value, event_wgt)
-    else:
-        histos[term]["tX+ttX"].Fill(value, event_wgt)
+    elif Sample_ID == -0624:
+	histos[term]["tW"].Fill(value, event_wgt) 
+    elif Sample_ID != -062500 and Sample_ID != -062524 and Sample_ID != -06062424 and Sample_ID != -999: #tHq, tHW, and ttWW 
+        histos[term]["others"].Fill(value, event_wgt)
+
+
+def FindBinNum(binning, value):
+    for i in range(1,len(binning)):
+	if value >= binning[i-1] and value < binning[i]:
+	    return i
+    #print len(binning)
+    #print "no bin covers this value %f" %value
+    return 0
+
+def GetSF(lep_type, pt, eta, lepMVA_cut):
```

Comment:
Is this function implemented in MakeHistos, at least for the L/M/T working points?